osm2gtfs/creators/stops_creator.py

```diff
@@ -48,14 +48,18 @@ def remove_unused_stops_from_feed(self, feed):
         from the final GTFS.
         It is called after the whole GTFS creation inside the main program.
         """
-        removed = 0
+        unused_stops = []
         for stop_id, stop in feed.stops.items():
             if stop.location_type == 0 and not stop.GetTrips(feed):
-                removed += 1
-                del feed.stops[stop_id]
-        if removed == 0:
+                #del feed.stops[stop_id]
+                unused_stops.append(stop_id)
+        if len(unused_stops) == 0:
             pass
-        elif removed == 1:
+        removed = 0
+        for stop_id in unused_stops:
+            removed += 1
```

Comment:
Could we use `len(unused_stops)` instead of this new variable `removed` ?

Reply:
sure, thanks!